clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -275,7 +281,7 @@ private void seekBlockInStream(long newPos) throws IOException {
           mCurrentCacheStream =
               mContext.getTachyonBlockStore().getOutStream(currentBlockId, -1, null);
         } catch (IOException ioe) {
-          // TODO(yupeng): Maybe debug log here.
+          LOG.info("Failed to get BufferedBlockOutStream: " + ioe.getMessage());
```

Comment:
A general comment for all the log outputs, it may be more informative to log at a less detailed level. For example `Failed to write to TachyonStore stream, block {blockId} will not be in TachyonStorage.` since the current message relies on the user knowing about the details of the code.